src/containers/Repo/Grader/GraderPage.js

```diff
@@ -219,11 +236,29 @@ export default class GraderPage extends Component {
                 </button>
 
                 <OverlayTrigger placement="bottom" overlay={ this.getVerificationTooltip() }>
-                  <button className="btn btn-primary" onClick={ this.handleVerification }>
+                  <button disabled={ submitting }
+                    className="btn btn-primary"
+                    onClick={ this.handleVerification }
+                  >
                     Verify Grades
                   </button>
                 </OverlayTrigger>
 
+                <i className={ 'btn fa fa-refresh' + (submitting ? ' fa-pulse disabled' : '') } />
```

Comment:
I think you're using the wrong icon. To hide icons while not loading it would be { loading && <LoadingIcon> } I might have a code that does this but if not add the icon from bootstrap